provision/fix-testdata.sh

```diff
@@ -0,0 +1,329 @@
+rm -rf /tmp/generated*
```

Comment:
don't commit this file ;-)

Reply:
ahh yeah, I was careful with that but seems like ended up adding it in multiple places. Resolving..